### PR TITLE
Sign in and sign up forms with ant

### PR DESCRIPTION
**Pull request by Simpcyclassy**

### CODE REVIEW

File: src/components/signin/components/Signin.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+import './style.css';
```
Comment (Way2nnadi) [inline src/components/signin/components/Signin.js]: move `import './style.css';` to `components\index.js`

File: src/components/signin/components/Signin.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  Placeholder_1,
+  Placeholder_2,
+  Reminder,
+  Title
+} from './constant';
+
+const FormItem = Form.Item;
+
+const SigninForm = Form.create()(
+  class extends React.Component {
```
Comment (Way2nnadi) [inline src/components/signin/components/Signin.js]: improper indent on line 27

File: src/components/signin/components/Signin.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    render() {
+
```
Comment (Way2nnadi) [inline src/components/signin/components/Signin.js]: remove unused empty line

File: src/components/signup/components/Signup.js [hunk 4]
```diff
@@ -0,0 +1,232 @@
+import React, { Component } from 'react';
+import './style.css';
```
Comment (Way2nnadi) [inline src/components/signup/components/Signup.js]: move `import './style.css';` to `components/index.js`

File: src/components/signup/components/Signup.js [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+  Label_3,
+  Login,
+  Read,
+  Title
+} from './constant';
+
+const FormItem = Form.Item;
+
+const SignupForm = Form.create()(
+  class extends React.Component {
```
Comment (Way2nnadi) [inline src/components/signup/components/Signup.js]: fix indent on line 37

File: src/components/signup/components/Signup.js [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+
```
Comment (Way2nnadi) [inline src/components/signup/components/Signup.js]: delete empty line

File: src/components/signup/components/Signup.js [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+      return (
+        <Modal
+          visible={visible}
+          title={Title}
+          onCancel={onCancel}
+          onOk={onCreate}
+        >
+          <Form 
+            layout="verticle"
+            className="signup_form"
```
Comment (Way2nnadi) [inline src/components/signup/components/Signup.js]: line 62-63, strings should be move to `constants.js`

File: src/components/signup/components/Signup.js [hunk 8]
```diff
@@ -0,0 +1,232 @@
...
+        <SignupForm
+          wrappedComponentRef={this.saveFormRef}
+          visible={this.state.visible}
+          onCancel={this.handleCancel}
+          onCreate={this.handleCreate}
+        />
+      </div>
+    );
+  }
+}
```
Comment (Way2nnadi) [inline src/components/signup/components/Signup.js]: move all the strings on this file to `constants.js`

File: src/components/signin/components/Signin.js [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+        <SigninForm
+          wrappedComponentRef={this.saveFormRef}
+          visible={this.state.visible}
+          onCancel={this.handleCancel}
+          onCreate={this.handleCreate}
+        />
+      </div>
+    );
+  }
+}
```
Comment (Way2nnadi) [inline src/components/signin/components/Signin.js]: move all the strings on this file to `constants.js`

File: src/components/signin/components/Signin.js [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+                            )}
+
+                            <Button 
+                                type={PRIMARY} 
+                                htmlType={SUBMIT} 
+                                shape={CIRCLE} 
+                                className={CLASSNAME_LOGIN}
+                            >
+                                {TITLE}
+                            </Button>
```
Comment (Way2nnadi) [inline src/components/signin/components/Signin.js]: delete this button, use the `okText` on the Modal instead
```
...
okText={SUBMIT}
...
```

File: src/components/signup/components/Signup.js [hunk 11]
```diff
@@ -0,0 +1,237 @@
...
+                        </FormItem>
+                        <FormItem>
+                            <Button
+                                type={PRIMARY}
+                                htmlType={SUBMIT}
+                                shape={CIRCLE}
+                                className={CLASSNAME_SIGNUP}
+                            > 
+                                {TITLE}
+                            </Button>
```
Comment (Way2nnadi) [inline src/components/signup/components/Signup.js]: delete this button, use the `okText` on the Modal instead
```
...
okText={TITLE}
...
```

File: src/components/signin/components/Signin.js (hunk 10 above)
Comment (Simpcyclassy) [reply src/components/signin/components/Signin.js]: Alright

File: src/components/signup/components/Signup.js (hunk 11 above)
Comment (Simpcyclassy) [reply src/components/signup/components/Signup.js]: OK

**Pull request merged**